packages/rancher-alerting-drivers/charts/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for rancher-alerting-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    # the registry where all images will be pulled from
+    systemDefaultRegistry: ""
+  preInstallHook:
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1000
+    image:
+      repository: rancher/rancher-agent
+      tag: v2.5.7 # TODO: change to v2.5.8
+      pullPolicy: IfNotPresent
+  # set this value if you want the sub-charts to be installed into
+  # a namespace rather than where this chart is installed
+  namespaceOverride: ""
+
+prom2teams:
```

Comment:
for both subcharts, just place all of these values in the subchart itself and only expose `enabled: false` at the top chart level. If its defined in the subchart's values.yaml, a user can always add it inside, but we don't want to expose them here unless necessary

Reply:
@aiyengar2 how does that actually work for global values ? Specifically curious about `global.cattle. systemDefaultRegistry`, if that doesn't exist at top level I don't know how UI knows to fill out sub-charts stuff. 
@jiaqiluo need to ensure we airgap test this.